angular-ngrx-scss/src/app/home/profile/org-profile/org-profile.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="profile-header">
+  <div class="grid grid-cols-12 container">
+    <app-org-info class="col-span-12"></app-org-info>
+    <app-profile-nav class="col-span-12"></app-profile-nav>
+  </div>
+</div>
+
+<div class="grid grid-cols-12 profile-body container">
+  <app-repo-controls class="col-span-12"></app-repo-controls>
+  <app-repo-list class="col-span-12" [repos$]="repos$"></app-repo-list>
```

Comment:
why don't we make "app-repo-list" a dumb component. Refactor that to use onpush for change detection.

Then, we pass data to it from this component like this "<app-repo-list class="col-span-12" [repos]="repos$ | async"></app-repo-list>"